jenkins/pipelines/ci/tikv/tikv_ghpr_coverage.groovy

```diff
@@ -64,12 +64,12 @@ stage("Cover") {
                 export FAIL_POINT=1
                 export ROCKSDB_SYS_SSE=1
                 export CARGO_INCREMENTAL=0
-                export RUSTFLAGS="-Zinstrument-coverage"
+                export RUSTFLAGS="-C instrument-coverage"
```

Comment:
What's the difference

Reply:
-Zinstrument-coverage is not supported anymore